pxl_scripts/px/redis_flow_graph/redis_flow_graph.pxl

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+''' Redis Overview Map
+Shows a graph of Redis requests in the cluster, with some latency information.
+'''
+
+import px
+
+ns_per_ms = 1000 * 1000
+ns_per_s = 1000 * ns_per_ms
+# Window size to use on time_ column for bucketing.
+window_ns = px.DurationNanos(10 * ns_per_s)
+
+
+def redis_flow_graph(start: str, ns: px.Namespace, from_entity_filter: str, to_entity_filter: str):
+
+    df = px.DataFrame('redis_events.beta', start_time=start)
+    df.pod = df.ctx['pod']
+    df.namespace = df.ctx['namespace']
+    df = df.drop(['upid'])
+
+    # Filter on namespace as specified by the user.
+    df = df[df.namespace == ns]
+
+    df.timestamp = px.bin(df.time_, window_ns)
+
+    # Aggregate the connections by timestamp.
+    df = df.groupby(['timestamp', 'remote_addr', 'pod']).agg(
+        latency_quantiles=('latency_ns', px.quantiles),
+        throughput_total=('latency_ns', px.count),
+    )
+    df.latency_p50 = px.DurationNanos(px.floor(px.pluck_float64(df.latency_quantiles, 'p50')))
+    df.latency_p90 = px.DurationNanos(px.floor(px.pluck_float64(df.latency_quantiles, 'p90')))
+    df.latency_p99 = px.DurationNanos(px.floor(px.pluck_float64(df.latency_quantiles, 'p99')))
+
+    # Specify to and from entities.
+    df = df[df.remote_addr != '']
+    df.to_entity = df.pod
+    df.from_entity_id = px.ip_to_pod_id(df.remote_addr)
+    df.from_entity = px.pod_id_to_pod_name(df.from_entity_id)
```

Comment:
style:     df.from_entity = px.pod_id_to_pod_name(px.ip_to_pod_id(df.remote_addr))

Avoids adding a temporary id to `df`